src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -128,7 +133,9 @@ export function getTypes(indices) {
 export function getIndices(includeAliases) {
   const ret = [];
   $.each(perIndexTypes, function (index) {
```

Comment:
nit: if you already touch that file, maybe we can use native methods instead?

Reply:
I was planning to include it with this task https://github.com/elastic/kibana/issues/75861 since they are related. WDYT?

Reply:
I think it's better to remove this all together as part of the getting rid of jquery task, otherwise we'll end up touching a lot of code that wasn't particularly related to this pr